checkmarx-ltd/VSTS-Plugin
# Plug 1936 - Fixes

**Changes**
Following changes done
- Space in description(Preset and Team info icon description).
- correct word ignore(Preset and Team info icon description).
- reverse all description related to SAST 9.7(SAST 9.7 is not yet released. So reverse the description like SAST 9.6 and below version).
- max value for retry waiting time for scan= 60 seconds

**Test Cases**
Case 1 
- Check Preset and Team icon description. (After full stop it should be space
- Spell correct in same(ignore)

Case 2 -
- wait time for retry scan for sast should be between 1 to 60. if its blank then it will take default value.
